src/apps/filters/h3.c

```diff
@@ -48,14 +48,19 @@ bool cellToLatLngCmd(int argc, char *argv[]) {
     Arg helpArg = ARG_HELP;
     DEFINE_CELL_ARG(cell, cellArg);
     Arg *args[] = {&cellToLatLngArg, &helpArg, &cellArg};
-    if (parseArgs(argc, argv, 3, args, &helpArg,
+    if (parseArgs(argc, argv, sizeof(args) / sizeof(Arg *), args, &helpArg,
                   "Convert an H3 cell to a latitude/longitude coordinate")) {
         return helpArg.found;
     }
     LatLng ll;
-    H3_EXPORT(cellToLatLng)(cell, &ll);
-    printf("%.10lf, %.10lf\n", H3_EXPORT(radsToDegs)(ll.lat),
-           H3_EXPORT(radsToDegs)(ll.lng));
+    H3Error err = H3_EXPORT(cellToLatLng)(cell, &ll);
+    if (err) {
+        return false;
```

Comment:
For my info, why return `true`/`false` from these functions? Wouldn't we just want to return the error code, so callers would get a `0` exit code on success and a meaningful non-zero code on failure?

Reply:
So, I *think* we started work on this binary before `H3Error` was a thing and it was doing `h3IsValid` checks on the inputs/outputs to determine if it should exit with an error or not.

Then the conversion of that was done this way because it was the most straightforward. Bubbling up the `H3Error` *does* make sense to me. If you don't mind, I'll keep that refactor for when I add in the next sub-command.